testing/path/common.py

```diff
@@ -186,9 +188,10 @@ def test_visit_norecurse(self, path1):
         assert "sampledir" in l
         assert not path1.sep.join(["sampledir", "otherfile"]) in l
 
-    def test_visit_filterfunc_is_string(self, path1):
+    @pytest.mark.paramtrize('fil', ['*dir', u'*dir'])
```

Comment:
Do you mean adding ``b'*dir'`` to the list?

About non ascii I'll see what I can do.